include/groonga/ii.h

```diff
@@ -68,6 +68,8 @@ grn_ii_buffer_commit(grn_ctx *ctx, grn_ii_buffer *ii_buffer);
 GRN_API grn_rc
 grn_ii_buffer_close(grn_ctx *ctx, grn_ii_buffer *ii_buffer);
 
+#define GRN_II_MAX_TF 0x00003
```

Comment:
This is temporary change during implementing it.
Because it's a little bit harder to prepare for the 130k tokens in the test.

Reply:
We can easily prepare for it using `generate-series` from `grntest`.